packages/node/test/e2e/cli-generate.spec.js

```diff
@@ -61,16 +49,6 @@ const DEP_FIXTURE_ENTRYPOINT_DIR = path.dirname(DEP_FIXTURE_ENTRYPOINT)
 
 const { testCLI } = createCLIMacros(test)
 
-test.beforeEach('setup temp dir', async (t) => {
-  t.context.tempdir = await mkdtemp(
-    path.join(tmpdir(), 'lavamoat-node-cli-test-')
-  )
-})
-
-test.afterEach('cleanup temp dir', async (t) => {
-  await rm(t.context.tempdir, { recursive: true, force: true })
-})
-
 test('"generate --help" prints help', testCLI, ['generate', '--help'])
```

Comment:
(pointless complaining, skill issue) I still find the macros syntax confusing. I had better results reading tests where test() was used in a loop

Reply:
as soon as v20 goes away we should switch. snapshots have been added in 22